topo_processor/util/aws_credentials.py

```diff
@@ -27,3 +31,18 @@ def get_credentials(bucket_name: str):
         default_credentials["secret_key"] = session_credentials.secret_key
         default_credentials["token"] = session_credentials.token
     return default_credentials
+
+
+def assumed_role_session(role_arn: str, base_session: botocore.session.Session = None) -> boto3.Session:
```

Comment:
This method is temporary. We need to discuss how we want to handle the authentication/role assumption in this project.